### ISSUE TITLE
Collection and Component Titles (and Abstracts) are inaccessible to screen readers

### ISSUE THREAD

**Issue by sdellis** [projectblacklight/arclight#943]
For optimal Accessibility, [`title` attributes should be avoided](https://a11yproject.com/posts/title-attributes/). Furthermore, there is no way to qualify the language used in a `title` attribute. Instead, the following example text should be wrapped in an element that has the appropriate `lang` attribute ISO code ('he' in this case). 

![arclight-inaccessible-example](https://user-images.githubusercontent.com/609368/66673980-cc7ef080-ec2f-11e9-8674-9c4bd30c721e.png)

[See/hear an example of the difference the `lang` attribute makes for those who use screen readers.](https://www.youtube.com/watch?v=ox5QVbZSPBk&feature=youtu.be)

**Comment by ggeisler** [comment]
Note that the `title` attribute would be removed to fix a previously reported issue (#930).

I'm not sure how feasible it is to accomplish the "wrapped in an element that has the appropriate lang attribute" aspect; a developer might need to look into whether the source EAD provides information about what language the element (scopenote or abstract, in this example) is written in.

**Comment by sdellis** [comment]
Ok, sounds good @ggeisler. EAD 3 provides [the `@lang` attribute](https://www.loc.gov/ead/EAD3taglib/index.html#attr-lang). 

Institutions may or may not use it, but it should probably do the right thing if they do. Princeton is currently working with the Google Translate API to identify and eventually update our foreign language content to use this attribute.

**Comment by seanaery** [comment]
Resolving this issue as part of a review of `arclight 1.0.x`. The `title` attribute mentioned is no longer present in v1. See issues #545 for multilingual description support and #553 for EAD3.
